build.rs

```diff
@@ -1,24 +1,37 @@
-use std::path::{Path, PathBuf};
+use std::path::{PathBuf, Path};
 
 fn main() -> std::io::Result<()> {
     println!("cargo:rerun-if-env-changed=LND_REPO_DIR");
-    let lnd_rpc_dir_owned;
-    let dir = match std::env::var_os("LND_REPO_DIR") {
+    let dir= match std::env::var_os("LND_REPO_DIR") {
         Some(lnd_repo_path) => {
             let mut lnd_rpc_dir = PathBuf::from(lnd_repo_path);
             lnd_rpc_dir.push("lnrpc");
-            lnd_rpc_dir_owned = lnd_rpc_dir;
-            &*lnd_rpc_dir_owned
+            lnd_rpc_dir.to_path_buf()
```

Comment:
I think this call shouldn't be needed. `lnd_rpc_dir` is already `PathBuf`. But wasting performance like this in a build script is not an issue.

Reply:
removed